packages/lesmis/src/views/DashboardView.js

```diff
@@ -7,17 +7,149 @@ import React from 'react';
 import styled from 'styled-components';
 import MuiContainer from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
+import { useUrlParams } from '../utils';
+import { useVitalsData } from '../api';
+import { EntityVitalsItem } from '../components';
 
-const TemplateBody = styled.section`
-  background: #e5e5e5;
-  padding-top: 1rem;
+const Wrapper = styled.section`
+  padding: 0, 0, 0, 0;
   min-height: 50vh;
+  background: #fbfbfb;
 `;
 
-export const DashboardView = () => (
-  <TemplateBody>
-    <MuiContainer>
-      <Typography>Dashboard View</Typography>
-    </MuiContainer>
-  </TemplateBody>
-);
+const Container = styled(MuiContainer)`
+  margin-left: 0;
+  padding: 0, 0, 0, 0;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 100%;
+`;
+
+const WrapContainer = styled(MuiContainer)`
+  padding-top: 0.5rem;
+  padding-left: 0;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  height: 100%;
+`;
+
+const TitleContainer = styled(MuiContainer)`
+  margin-left: 0;
+  padding-left: 0;
+  width: 100%;
+`;
+
+const RedTitle = styled(Typography)`
+  color: ${props => props.theme.palette.primary.main};
+`;
+
+export const DashboardView = () => {
+  const { entityCode } = useUrlParams();
+  const vitals = useVitalsData(entityCode);
+  switch (vitals.type) {
+    case 'district':
+      // "Province" in lesmis context
+      return (
+        <Wrapper>
+          <Container>
+            <WrapContainer>
+              <TitleContainer>
+                <RedTitle variant="h4">Province Profile:</RedTitle>
+              </TitleContainer>
+              <EntityVitalsItem name="Province Code" value={vitals.code} icon="PinDrop" />
```

Comment:
Nice use of entity vitals component. Seems like a good design of component that is flexible and re-usable.